pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -4461,6 +4461,60 @@ func createJvmTestObjectWithoutJavaMaxMemRatio() *api.JvmObject {
 	return &jvmObject
 }
 
+func TestCPUDefaultLimit(t *testing.T) {
```

Comment:
Can you please also add these test scenario:
1) Test case when are set CPU Limits (not default values)
2) Set CPU Limits in different form - e.g. instead of ms use cores (2000m = 2)
3) Test case when are set at the same time CPU Limits and Object's Resources limits with different values
4) Test case when is set Object's Resources request higher then CPU Limit (no Resource limit)